moped-editor/src/index.js

```diff
@@ -48,7 +48,7 @@ Amplify.configure({
 });
 
 ReactDOM.render(
-  <BrowserRouter basename={"/moped"}>
+  <BrowserRouter>
```

Comment:
it seems that in react-router `6.0.0-beta.0`, the `basename` prop was being ignored. that's fixed, and so
'moped' was being duplicated in all route paths.

it would seem more elegant to use the `basepath` prop and get the "/moped" cruft out of our paths. i did run with that and get things working with that approach. however, if we go with the `basename` approach, it's not possible to redirect the root path `/` to `/moped`, which means that in local dev and deploy previews you'd have to manually append "/moped" to your base URL. anyway, if that sounds preferable to y'all it's a minor code change.

Reply:
Might be more elegant but I think explicit routes can be nice too since we had some trouble tracking down what was going on with the previous config the other day.